package.json

```diff
@@ -15,7 +15,9 @@
     "rest": "npx codeceptjs run --grep '@restAPI'",
     "webdriver": "npx codeceptjs run --grep '@webDriver'",
     "janus": "npx codeceptjs run --grep '@JanusBeta'",
-    "prodSafe": "npx codeceptjs run --grep '@prodSafe'"
+    "prodSafe": "npx codeceptjs run --grep '@prodSafe'",
+    "bdd-rest": "npx codeceptjs run --features --grep '@restAPI'",
```

Comment:
I think you meant to expand @prodSafe to be @productionSafe (as you did in the test)

Reply:
this is addressed in the latest commit